Mathlib/NumberTheory/Rayleigh.lean

```diff
@@ -162,29 +165,25 @@ theorem rayleigh_pos {r s : ℝ} (hrs : r.IsConjugateExponent s) :
     have hj := Int.ceil_pos.1 (lt_trans zero_lt_one hj)
     have := pos_of_mul_pos_left hj hs
     rwa [Int.cast_pos] at this
-  rw [Set.mem_symmDiff, hb₁ _ hrs.nonneg, hb₂ _ hrs.symm.nonneg, ← rayleigh_compl hrs,
+  rw [Set.mem_symmDiff, hb₁ _ hrs.nonneg, hb₂ _ hrs.symm.nonneg, ← compl_beattySequence hrs,
     Set.not_mem_compl_iff, Set.mem_compl_iff, and_self, and_self]
   exact or_not
 
+theorem rayleigh_pos' {r s : ℝ} (hrs : r.IsConjugateExponent s) :
+    {beattySequence' r k | k > 0} ∆ {beattySequence s k | k > 0} = {n | 0 < n} := by
+  rw [symmDiff_comm, rayleigh_pos hrs.symm]
+
+/-- Let `r` be an irrational number. Then `B⁺_r` and `B⁺'_r` are equal. -/
+theorem irr_beattySequence_pos_eq {r : ℝ} (hr : Irrational r) :
+    {beattySequence r k | k > 0} = {beattySequence' r k | k > 0} := by
```

Comment:
I would state it the other way, from the assumption that `beattySequence` is simpler than `beattySequence'`
```suggestion
    {beattySequence' r k | k > 0} = {beattySequence r k | k > 0} := by
```

Reply:
Done. `beattySequence` is simpler.